Header header
uint32 item_uid
uint32 field_uid
float32 weight